functions/src/api/v1/invite/inviteParticipants.ts

```diff
@@ -0,0 +1,144 @@
+ *     tags:
+ *       - rooms
+ *     consumes:
+ *     - multipart/form-data
+ *     produces:
+ *     - application/json
+ *     parameters:
+ *       - name: hostname
+ *         description: The name or organisation which sent the invite(s)
+ *         in: x-www-form-urlencoded
+ *         required: true
+ *         type: string
+ *       - name: destinations
+ *         description: An array of destinations
+ *         in: x-www-form-urlencoded
+ *         required: true
+ *         examples:
+ *            mixedTypeAndLang:
+ *                summary: Mixed email, sms and languages
+ *                value: [{email: "user@example.com", lang: "en"}, {phone: "+33600000000", lang:"fr"}, {phone: "+33600000000", lang:"fr", country:"en"}]
+ *            emailInvite:
+ *                summary: One email invite with French lang
+ *                value: [{email: "user@example.com", lang: "fr"}]
+ *         schema:
+ *            type: array
+ *            items:
+ *                type: object
+ *                properties:
+ *                   email:
+ *                       type: string
+ *                   phone:
+ *                       type: string
+ *                   lang:
+ *                       type: string
+ *                       default: en
+ *                       enum:
+ *                           - en
+ *                           - fr
+ *                           - de
+ *                           - es
+ *                           - gr
+ *                           - hu
+ *                           - it
+ *                           - ro
+ *
+ *     responses:
+ *       200:
+ *         description: One or many invites were sent successfully. It will return a 2xx if some invitation where not able to be sent (malformed email or phone number for eg).
+ *         content:
+ *           application/json:
+ *             schema:
+ *               example: {
+ *                   emailsSent: ["participant@example.com", "anotherParticipant@example.com"],
+ *                   smsSent: ["+33600000000"],
+ *               }
+ *       400:
+ *         description: request content (x-www-form-urlencoded) not correct, or zero invitation delivered
+ *       401:
+ *         description: missing authorization bearer token
+ *       403:
+ *         description: token not valid
+ *       404:
+ *         description: room does not exist
+ *       412:
+ *         description: authorization header present but not formatted correctly
+ */
+export const inviteParticipants = wrap(async (req: Request, res: Response) => {
+    const roomId = req.params.roomId
```

Comment:
Destructuring has a cost for executing, memory and also reading (in my case a least). 
It's useful when needed (line 88 for eg), not in this case

Reply:
cost in performance you mean ? I think performance cost is negligible and readability is increased since the line is shorter

```js
const {roomId} = req.params
```

and there is no `roomId` repetition on both sides of the expression. Of course the benefits are clearer when the destructuring includes more than one property, or when the property is used in multiple spots (avoids code duplication)